src/header.tsx

```diff
@@ -173,47 +174,63 @@ export const HeaderLayout: React.FC<HeaderLayoutProps> = ({
     extra,
     sideBySideExtraOnDesktop = false,
     dataAttributes,
+    bleed = false,
 }) => {
     const {isTabletOrSmaller} = useScreenSize();
 
     return (
-        <ResponsiveLayout
-            isInverse={isInverse}
-            dataAttributes={{'component-name': 'HeaderLayout', ...dataAttributes}}
-        >
-            <OverscrollColor />
-            {isTabletOrSmaller ? (
-                <Box paddingTop={header ? 32 : 0} paddingBottom={24}>
-                    <Stack space={24}>
-                        {header}
-                        {extra}
-                    </Stack>
-                </Box>
-            ) : sideBySideExtraOnDesktop ? (
-                <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={48}>
-                    <GridLayout
-                        template="6+6"
-                        left={
+        <div>
```

Comment:
I think you can replace this `<div>` with a fragment (`<>`)

Reply:
Fixed!